app/tests/test_queries.py

```diff
@@ -262,348 +261,20 @@ def test_build_fuzzy_search_query_no_results(self, client: FlaskClient):
-                "test series3",
-                "01/01/2023",
-                1,
-                1,
-            ),
-        ]
-
-    def test_browse_with_date_from_and_to_filter(self, client: FlaskClient):
-        """
-        Given multiple File objects in the database and consignment transfer complete date
-            that matches all of them
-        When browse_data is called with date_from and date_to
-        Then it returns a list containing multiple dictionary for the matching record with
-            date in between the range
-        """
-        files = create_multiple_test_records()
-
-        filters = {
-            "date_range": {"date_from": "01/02/2023", "date_to": "28/02/2023"}
-        }
-        result = browse_data(page=1, per_page=per_page, filters=filters)
-
-        assert result.items == [
-            (
-                files[4].consignment.series.body.BodyId,
-                "testing body5",
-                files[4].consignment.series.SeriesId,
-                "test series5",
-                "15/02/2023",
-                1,
-                1,
-            ),
-            (
-                files[5].consignment.series.body.BodyId,
-                "testing body6",
-                files[5].consignment.series.SeriesId,
-                "test series6",
-                "15/02/2023",
-                1,
-                1,
-            ),
-            (
-                files[6].consignment.series.body.BodyId,
-                "testing body7",
-                files[6].consignment.series.SeriesId,
-                "test series7",
-                "15/02/2023",
-                1,
-                1,
-            ),
-            (
-                files[7].consignment.series.body.BodyId,
-                "testing body8",
-                files[7].consignment.series.SeriesId,
-                "test series8",
-                "15/02/2023",
-                1,
-                1,
-            ),
-            (
-                files[8].consignment.series.body.BodyId,
-                "testing body9",
-                files[8].consignment.series.SeriesId,
-                "test series9",
-                "15/02/2023",
-                1,
-                1,
-            ),
-        ]
-
-    def test_browse_with_date_from_and_to_filter_no_result(
-        self, client: FlaskClient
-    ):
-        """
-        Given multiple File objects in the database and consignment transfer complete date
-            that matches all of them
-        When browse_data is called with date_from and date_to
-        Then if date range not matched it returns empty list containing multiple dictionary
-        """
-        create_multiple_test_records()
-
-        filters = {
-            "date_range": {"date_from": "01/03/2023", "date_to": "31/05/2023"}
-        }
-        result = browse_data(page=1, per_page=per_page, filters=filters)
-
-        assert result.items == []
-
-    def test_browse_with_transferring_body_sorting_a_to_z(
```

Comment:
github's PR diff of the changes ended up being poor due to the similar tests :( so looks like there's lots of new tests which are in fact these tests incorrectly marked as changed.

Reply:
I have ended up skimming over the tests and they generally look okay but I wouldnt fully trust it, because there were a lot of lines changed and so I may have missed some things

Reply:
without filter and with filter are new test cases added for this ticket, rest of sorting ones were done with sorting tickets